code/modules/projectiles/projectile/bullets/rifle.dm

```diff
@@ -39,55 +39,55 @@ heavy rifle calibers (12.7, 14mm, 7.62): Uranium, Contaminated, Incin
 
 /obj/item/projectile/bullet/a556
 	name = "5.56 FMJ bullet"
-	damage = 0
-	wound_bonus = 15
-	bare_wound_bonus = -18
+	damage = 32
+	wound_bonus = 24
+	bare_wound_bonus = 18
 
 /obj/item/projectile/bullet/a556/match
 	name = "5.56 match bullet"
-	damage = 29
+	damage = 28
 	armour_penetration = 0.21
-	wound_bonus = 15
-	bare_wound_bonus = -16
+	wound_bonus = 18
+	bare_wound_bonus = 12
```

Comment:
```suggestion
	bare_wound_bonus = 0
```
Since 5.56 match's main gimmick is armor penetration and speed, it should do little to unarmored targets